src/main.rs

```diff
@@ -46,9 +49,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     println!("SnapchainService (ID: {}) listening on {}", args.id, addr);
 
-    let _ = tracing_subscriber::fmt()
-        .with_env_filter(EnvFilter::from_default_env())
-        .try_init();
+    let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info"));
+
+    tracing_subscriber::fmt()
+        .json()
```

Comment:
JSON is important for prod, but we might want to use simple text for the dev env or until we're ready for testnet. We're going to be staring at a lot of log lines until then, and it's worth it to make it easy to read.

Reply:
Sounds good, let's add an config switch for logging format as soon as we have a config system.